pkgs/development/ocaml-modules/cstruct/default.nix

```diff
@@ -1,23 +1,20 @@
-{ lib, fetchurl, buildDunePackage, bigarray-compat, alcotest, ocaml }:
+{ lib, fetchurl, buildDunePackage, fmt, alcotest, crowbar }:
 
 buildDunePackage rec {
   pname = "cstruct";
-  version = "6.0.1";
+  version = "6.1.0";
```

Comment:
6.1.1 is out.
https://github.com/mirage/ocaml-cstruct/releases/tag/v6.1.1
Any particular reason to use 6.1.0?